### ISSUE TITLE
Panic when workspace member name has multiple slashes

### ISSUE THREAD

**Issue by treefitty** [denoland/deno#26334]
I want to create a simple monorepo structure that mimics the type of setup I have with Nx.dev using Deno2.0 workspaces.
However when I created a simple project under `/libs` the compiler threw a panick

I have a reproducible here: https://github.com/treefitty/deno-test

Clone and run `deno run main.ts`

```
============================================================
Deno has panicked. This is a bug in Deno. Please report this
at https://github.com/denoland/deno/issues/new.
If you can reliably reproduce this panic, include the
reproduction steps and re-run with the RUST_BACKTRACE=1 env
var set and include the backtrace in your report.

Platform: macos x86_64
Version: 2.0.0
Args: ["deno", "run", "main.ts"]

thread 'main' panicked at /Users/brew/Library/Caches/Homebrew/cargo_cache/registry/src/index.crates.io-6f17d22bba15001f/deno_config-0.37.1/src/workspace/resolver.rs:564:16:
called `Result::unwrap()` on an `Err` value: InvalidPathWithVersion(PackageReqReferenceInvalidWithVersionParseError { kind: Jsr, current: "@deno-test/libs/math@^0.1.0/", suggested: "@deno-test/libs@^0.1.0//math" })
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

**Comment by dsherret** [comment]
Change the name from `"name": "@deno-test/libs/math"` to something with only a single slash `"name": "@deno-test/libs-math"

**Comment by dsherret** [comment]
> So is this the suggested work around and you're not going to support directory structures with workspaces, or a temp work around until a patch lands?

Package names can only have one slash in them. Once it has another slash then that's the export of the package. You can keep the same file structure, but you just have to give it a name in the form `@scope/name` in the deno.json.

**Comment by treefitty** [comment]
No worries, cheers

**Comment by dsherret** [comment]
I'm going to reopen because it's a bug for it to panic. It should give a descriptive error.

**Comment by satyarohith** [comment]
Fixed by the deno_config upgrade in #26568
